tests/lib/rules/no-ember-data-global-imports.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-ember-data-global-imports');
+const RuleTester = require('eslint').RuleTester;
+const parserOptions = { ecmaVersion: 6, sourceType: 'module' };
+
+const message = 'Imports from @ember-data packages should be prefered over imports from ember-data';
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
```

Comment:
Use `new RuleTester(parserOptions)` instead of passing `parserOptions` to each test case.

Reply:
nice